internals/build.rs

```diff
@@ -51,7 +51,7 @@ fn write_macro(mut macro_file: impl io::Write, msrv_minor: u64, minor: u64) -> i
     writeln!(macro_file, "/// Example:")?;
     writeln!(macro_file, "/// ```")?;
     writeln!(macro_file, "/// bitcoin_internals::rust_version! {{")?;
-    writeln!(macro_file, "///     if >= 1.70 {{")?;
+    writeln!(macro_file, "///     if >= 1.78 {{")?;
```

Comment:
I don't understand the reason for this change? Also the lines below need updating as well.

In 84a9df10dd648eb728a4a6ee8d026f576ccff704

Reply:
Updating in what way?

And the reason is that the `rust_version!` macro doesn't compile if you give it rust versions before our MSRV. It has an "unknown Rust version" panic case.

Reply:
Woops context is dropped. There are output messages that have the 1.70 in them. Why 1.78 and not 1.74?

Reply:
Because 1.74 is the MSRV. Anything "conditionally compiled on 1.74" will be unconditionally compiled.

Reply:
Where does 1.78 come from?

Reply:
Don't mind me.